setup.py

```diff
@@ -24,6 +24,25 @@
     long_description_content_type="text/markdown",
     url="https://github.com/SymbiFlow/prjxray",
     packages=['prjxray'],
+    install_requires=[
+        'sdf-timing @ git+https://github.com/symbiflow/python-sdf-timing',
+        'fasm @ git+https://github.com/symbiflow/fasm',
+        'intervaltree',
+        'junit-xml',
```

Comment:
junit-xml is definitely not needed in setup.py?

Reply:
removed